kura/distrib/src/main/resources/intel-up2-ubuntu-18/kura_install.sh

```diff
@@ -88,6 +93,7 @@ mkdir -p /var/named
 chown -R bind /var/named
 cp ${INSTALL_DIR}/kura/install/named.ca /var/named/
 cp ${INSTALL_DIR}/kura/install/named.rfc1912.zones /etc/
+cp ${INSTALL_DIR}/kura/install/usr.sbin.named /etc/apparmor.d/
```

Comment:
Named is used for configuring DNS servers. I really don't know if it is needed anymore or not...

Reply:
me neither...